sale_timesheet_order_line_no_update/models/project.py

```diff
@@ -43,6 +43,19 @@ def _compute_sale_line_id_domain(self):
                 ("company_id", "=", rec.company_id.id),
             ]
 
-    @api.onchange("sale_line_id")
-    def _onchange_sale_line_id(self):
-        self.task_ids.write({"new_sale_line_id": self.sale_line_id})
+    def update_tasks_default_sol(self):
+        for rec in self:
+            rec.task_ids.new_sale_line_id = rec.sale_line_id
+
+    @api.model
```

Comment:
This is not `api.multi_create`. I really think you should go with the computed writable field.

Reply:
That was the first solution i tried. But it breaks odoo standard tests in sale_timesheet.